cmd/containerd-shim-runhcs-v1/task_jobobject_update.go

```diff
@@ -0,0 +1,53 @@
+package main
```

Comment:
This file will be going away once you rebase on latest master right?

Reply:
Support for updating CPU will be in vb, but I was thinking about keeping this for pre-vb versions. What do you think?

Reply:
Hmm, I am confused, in the PR description you mentioned that this file is temporary until #903 is merged so I thought this won't be needed anymore. But looks like we still need this file for pre-vb?

Reply:
I decided to take this out.